### PR TITLE
release 1.0.0-prerelease.11

### PR DESCRIPTION
**Pull request by mwcz**
This merges in the version number updates from release 1.0.0-prerelease.11

### CODE REVIEW

File: elements/pfelement/reveal.js [hunk 1]
```diff
@@ -1,51 +0,0 @@
-/*
```
Comment (starryeyez024) [inline elements/pfelement/reveal.js]: @mwcz did we intend to remove reveal.js ?

Comment (mwcz) [reply elements/pfelement/reveal.js]: Yes, it's a compiled asset that shouldn't have been there to begin with.  @castastrophe noticed that it was a stowaway. :)

Comment (mwcz) [reply elements/pfelement/reveal.js]: Oh and it isn't gone for good, it lives in `elements/pfelement/src/reveal.js`.  This just removed the compiled version.

Comment (starryeyez024) [reply elements/pfelement/reveal.js]: Listen, Gandalf Taps Musically  👍

**Pull request merged**